.changeset/odd-bobcats-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-openmrs': minor
+---
+
+Implement namespaced http generic helpers
```

Comment:
Can we be more accurate here? What we've actually done is add  a new `http.request()` operation. I would not call that a `helper`.

It is probably worth ALSO adding a helpful note to explain the difference between `fetch()` and `get()` (the difference being that `fetch` requests against the instanceURL directly and does not append anything to the path).